src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -178,10 +178,7 @@
                 // be not strict there
                 "METHOD_DEF", "CTOR_DEF", "CLASS_DEF", "ENUM_DEF", "INTERFACE_DEF")
                 .collect(Collectors.toSet()));
-        GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("SeparatorWrap", Stream.of(
-                // state of configuration until
-                // https://github.com/checkstyle/checkstyle/issues/3752
-                "RBRACK", "AT", "ELLIPSIS", "SEMI", "ARRAY_DECLARATOR",
+        GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("SeparatorWrap", Stream.of("RBRACK", "AT", "SEMI",
```

Comment:
Same as https://github.com/checkstyle/checkstyle/pull/4580#discussion_r125167696.
These tokens in this test need a reason why they are staying as ignored.